x-pack/plugins/observability_solution/infra/server/lib/alerting/register_rule_types.ts

```diff
@@ -20,6 +20,7 @@ export const LogsRulesTypeAlertDefinition: IRuleTypeAlerts = {
   mappings: { fieldMap: legacyExperimentalFieldMap },
   useEcs: true,
   useLegacyAlerts: true,
+  shouldWrite: false,
```

Comment:
This definition looks like it's only used for the log threshold rule type, so you should be able to set `shouldWrite: true` here and remove the override in `register_log_threshold_rule_type`

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/178680/commits/caa163f3d5455b6a09160abc491d9a870fcd208f